docs/montecarlo/sourceintegration.rst

```diff
@@ -0,0 +1,112 @@
+********************************
+Direct integration of the radiation field
+********************************
+
+.. note::
+
+  The current implementation only works with the downbranch line interaction scheme.
+
+
+:cite:`Lucy1999a` describes an alternative method for the generation of spectram for the supernova ejecta. Instead of using the frequency and energy of virtual montecarlo packets to create a spectrum through binning, one can use estimators collected during the montecarlo simulation to formally integrate the radiation field. Spectra generated using this method do not contain montecarlo noise directly. Here the monte carlo nature of the simulation only affects the strengths of lines and thus the spectra appear to be of better quality.
```

Comment:
Important: The description should reflect that the method formally integrates over the line source functions and that these are determined during the simulation using volume-based estimators.